tools/stress-cluster/chaos/README.md

```diff
@@ -214,6 +214,7 @@ The following environment variables are currently populated by default into the
 
 ```
 AZURE_CLIENT_ID=<value>
+AZURE_CLIENT_OID=<value>
```

Comment:
Where was this one added? is it now in keyvaults by default?

Reply:
Yeah it's been there a while, so updating the docs for parity.